### ISSUE TITLE
[Documentation]: FAQ, can badgerdb accessible by 2 program at the same time?

### ISSUE THREAD

**Issue by kokizzu** [dgraph-io/badger#1823]
### What version of Badger is the target?

v2

### Documentation.

FAQ whether badgerdb can be accessed by 2 program at the same time? (1 program writing, 1 program reading) or there's risk doing this?

**Comment by matthewmcneely** [comment]
No, check out https://dgraph.io/docs/badger/get-started/#opening-a-database

Specifically the line that reads "Please note that Badger obtains a lock on the directories so multiple processes cannot open the same database at the same time."
